src/utils.cpp

```diff
@@ -393,8 +391,24 @@ QuantLib::DayCounter getDayCounter(const double n){
         return QuantLib::ActualActual(QuantLib::ActualActual::Historical);
     else if (n==12)
         return QuantLib::ActualActual(QuantLib::ActualActual::AFB);
-    else // if (n==13)
+    else if (n==13)
         return QuantLib::ActualActual(QuantLib::ActualActual::Euro);
+    else if (n==14)
+        return QuantLib::Thirty360(QuantLib::Thirty360::USA);
+    else if (n==15)
+        return QuantLib::Thirty360(QuantLib::Thirty360::BondBasis);
+    else if (n==16)
+        return QuantLib::Thirty360(QuantLib::Thirty360::European);
+    else if (n==17)
+        return QuantLib::Thirty360(QuantLib::Thirty360::EurobondBasis);
+    else if (n==18)
+        return QuantLib::Thirty360(QuantLib::Thirty360::Italian);
+    else if (n==19)
+        return QuantLib::Thirty360(QuantLib::Thirty360::German);
+    else
+        // Stop on verbose error.
+        // Do not silently default to the arbitrarily last else statement because it can conceal bugs in user code.
+        throw std::range_error("Unknown option " + std::to_string(n)); // is this the right way to cast double to string?
```

Comment:
We can use `Rcpp::stop("Unknown option %d", n);` which is more idiomatic.  

Rest looks good at an initial glance, I will come back tomorrow.

Reply:
I just took care of this.